halibot/halibot.py

```diff
@@ -130,10 +136,7 @@ def _instantiate_modules(self):
 			self.add_module_instance(k, obj(self, conf))
```

Comment:
No, since packages (and probably sooner) the loading logic is completely the same now. Will merge these functions in the next revision.